gapii/cc/BUILD.bazel

```diff
@@ -28,8 +28,16 @@ apic_template(
     ],
 )
 
+apic_compile(
+    name = "gles_compiled",
+    api = "//gapis/api/gles:api", # TODO: Let apic handle multiple api files in one compile.
+#    optimize = True,
```

Comment:
I would assume we actually DO want these on when building -opt?

Reply:
One step at a time...

Reply:
Sounds good. It would be good to re-enable them in a followup.

Reply:
You were right in that disabling the inlining pass makes a semi-opt build perform fast enough.